frontend/sw360-portlet/src/main/webapp/html/projects/includes/projects/eccStatus.jspf

```diff
@@ -0,0 +1,100 @@
+<%--
+  ~ Copyright Siemens AG, 2017. Part of the SW360 Portal Project.
+  ~
+  ~ SPDX-License-Identifier: EPL-1.0
+  ~
+  ~ All rights reserved. This program and the accompanying materials
+  ~ are made available under the terms of the Eclipse Public License v1.0
+  ~ which accompanies this distribution, and is available at
+  ~ http://www.eclipse.org/legal/epl-v10.html
+--%>
+
+<core_rt:if test="${releasesAndProjects.size() == 0 }">
+    <h3>Please add linked releases or projects with linked releases to view their ECC status here.</h3>
+</core_rt:if>
+
+<core_rt:if test="${releasesAndProjects.size() > 0 }">
+    <div id="releasesECCTableDiv">
+        <table id="releasesECCTable" cellpadding="0" cellspacing="0" border="0" class="display">
+            <tfoot>
+            <tr>
+                <th width="10%"></th>
+                <th width="20%"></th>
+                <th width="10%"></th>
+                <th width="10%"></th>
+                <th width="20%"></th>
+                <th width="20%"></th>
+                <th width="10%"></th>
+            </tr>
+            </tfoot>
+        </table>
+        <core_rt:if test="${inProjectDetailsContext}">
+            <div class="clear-float"></div>
+            <span class="pull-right">
+                <input type="button" class="addButton" id="exportReleasesSpreadsheetECCButton" value="Export Spreadsheet"/>
```

Comment:
I get an Internal Server Error when clicking this button, cause by a "NullPointerException" while generating the XLSX. It seems to be due to a missing font. Does anything needs to be configured?

Reply:
I cannot reproduce this behavior.
Alex and me could execute the export spreadsheet function successfully.
@mcjaeger please verify the export spreadsheet function on both pagaes (clearing status / ecc status)

Reply:
no, this null pointer excpetion I did not see. Generation works fine for me.